### ISSUE TITLE
iPhone X layout

### ISSUE THREAD

**Issue by steamGerbil** [AlexLittlejohn/ALCameraViewController#247]
When using this on the new iPhone X the flash button is in the unsafe zone at the top of the screen. Can the layout be updated to use the safe guide in iOS11?

**Comment by steamGerbil** [comment]
It looks like this can be resolved as follows. Please check as I'm new to coding

```
if #available(iOS 11, *) {
            flashButtonEdgeConstraint = NSLayoutConstraint(
                item: flashButton,
                attribute: attribute,
                relatedBy: .equal,
                toItem: view.safeAreaLayoutGuide,
                attribute: attribute,
                multiplier: 1.0,
                constant: constraintRight ? 8 : -8)
            view.addConstraint(flashButtonEdgeConstraint!)
        } else {
            flashButtonEdgeConstraint = NSLayoutConstraint(
                item: flashButton,
                attribute: attribute,
                relatedBy: .equal,
                toItem: view,
                attribute: attribute,
                multiplier: 1.0,
                constant: constraintRight ? 8 : -8)
            view.addConstraint(flashButtonEdgeConstraint!)
        }
```

**Comment by AlexLittlejohn** [comment]
Fixed in 3.0.3
